### ISSUE TITLE
definition is displayed twice in EFO

### ISSUE THREAD

**Issue by nicolevasilevsky** [EBISPOT/efo#730]
https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0005223

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky 
Yes, one definition is EFO's, and the other is Mondo's that Mondo took from EFO...
We're aware of this issue (see https://github.com/EBISPOT/efo/issues/420). 
Also, we have a similar issue with synonyms (https://github.com/EBISPOT/efo/issues/276#issuecomment-585722352). 
We'll see if we can up the priority, but I'm afraid it won't be immediate. Thanks for mentioning this!

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky 
Update: @zoependlington may be able to look into a fix for this in a couple of weeks.

**Comment by nicolevasilevsky** [comment]
Thanks! It isn't an issue for me, you don't need to prioritize it if other things are more pressing. :) And feel free to close this ticket, since it is a duplicate.

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26):** this appears resolved and closable.

- The reported term, [EFO_0005223 "acute stress reaction"](https://www.ebi.ac.uk/ols4/ontologies/efo/classes?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0005223), has since been obsoleted (EFO 3.88.0) and replaced by MONDO:0003763. On OLS4 today, both the obsolete term and its Mondo replacement each display a **single** definition — the EFO-vs-Mondo double described here is gone.
- The root cause was tracked in #420 ("Multiple labels/definitions"), closed in Feb 2022, with a follow-up cleanup in #2290, closed in Nov 2024. The reporter also noted at the time that this ticket could be closed as a duplicate of #420.
- The legacy `efo:definition` annotation property is declared but no longer used on any term in `efo-edit.owl`, so that duplication pathway is also closed off.

One residual note for maintainers: a scan of `efo-edit.owl` still finds ~61 **non-obsolete** EFO terms carrying more than one `IAO_0000115` definition (e.g. EFO_0000266 "aortic stenosis" currently shows three near-identical definitions on OLS4). If that tail is worth cleaning up, it may deserve a fresh, scoped issue rather than keeping this 2020 ticket open.

**Recommendation:** close as resolved (duplicate of the already-closed #420), optionally spinning off a new issue for the remaining multi-definition terms.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
